ledfx/virtuals.py

```
@@ -1082,7 +1103,7 @@ def config(self, new_config):
                     ):
                         self._active_effect.clear_melbank_freq_props()
 
-                    if _config["rows"] != self._config["rows"]:
+                    if _config["rows"] != self.rows:
                         if hasattr(self._active_effect, "set_init"):
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)